.jenkins/pytorch/test.sh

```diff
@@ -56,9 +56,7 @@ if [[ "$BUILD_ENVIRONMENT" != *ppc64le* ]]; then
   pip_install --user mypy || true
 fi
 
-if [[ $PYTHON_VERSION == "2" ]]; then
```

Comment:
Okay I lied, this is the one place where we use it. But theoretically we can just install it always